Sources/Internal/LayoutItemTypeEnumerator.swift

```diff
@@ -102,6 +103,10 @@ final class LayoutItemTypeEnumerator {
       } else {
         return .day(calendar.day(byAddingDays: -1, to: day))
       }
+    
+    case .monthFooter(let month):
+        let day = calendar.day(containing: calendar.lastDate(of: month))
+        return .day(day)
```

Comment:
Ah, some more 4 space instead of 2 space indentation. I'll look into enforcing this at the project level. This is probably annoying to deal with, sorry!

Reply:
It's okay! As for enforcing, can look into `swiftlint` or something similar?

Reply:
yeah, great idea. We already use that for the Airbnb app repo internally, and it pulls rules from our Swift style guide: https://github.com/airbnb/swift